src/Aspire.Dashboard/Components/Controls/SummaryDetailsView.razor.cs

```diff
@@ -17,7 +17,7 @@ public partial class SummaryDetailsView<T> : IGlobalKeydownListener, IDisposable
     public RenderFragment? Summary { get; set; }
 
     [Parameter]
-    public RenderFragment? Details { get; set; }
+    public RenderFragment<T>? Details { get; set; }
```

Comment:
As summary details view uses the same selected item, we can just pass it to the render fragment